websitev2/docs/advanced/custom-activation-strategy.md

```diff
@@ -40,7 +40,7 @@ class TimeStampStrategy extends Strategy {
   }
 
   isEnabled(parameters, context) {
-    Date.parse(parameters.enableAfter) < Date.now();
+    Date.parse(parameters.enableAfter) >= Date.now();
```

Comment:
The problem here is simply the missing `return. 

`enabledAfter` means that the feature toggle should be considered `enabled` if the current point in time is higher than the configured parameter. 

Example:
*enabledAfter* = '2021-08-19' should enabled the feature toggle when current time is higher than 2021-08-19. This will happen at 2021:08-19 00:01.

Reply:
Damn; you're plain right.

Reply:
I just edited it again; should be ready to roll.

Reply:
thanks! 👍🏼